solidity/contracts/helpers/TestLiquidityPoolV1Converter.sol

```diff
@@ -12,19 +13,15 @@ contract TestLiquidityPoolV1Converter is LiquidityPoolV1Converter {
         uint32 _maxConversionFee
     ) public LiquidityPoolV1Converter(_token, _registry, _maxConversionFee) {}
 
-    function time() internal view override returns (uint256) {
-        return currentTime;
-    }
-
-    function setTime(uint256 _currentTime) public {
-        currentTime = _currentTime;
-    }
-
     function removeLiquidityTest(
         uint256 _amount,
         IERC20Token[] memory _reserveTokens,
         uint256[] memory _reserveMinReturnAmounts
     ) public {
         reserveAmountsRemoved = removeLiquidity(_amount, _reserveTokens, _reserveMinReturnAmounts);
     }
+
+    function time() public view virtual override(Time, TestTime) returns (uint256) {
```

Comment:
I wonder if there's an OO pattern which would allow us to get rid of this function in this contract.
After all, it is already declared in contract `TestTime`, which this contract inherits from.
I guess that there might be a diamond issue here, because this contract also inherits from contract `Time` which also implements this function...

Reply:
Yeah. I couldn't find an easier solution :(